packages/elements/src/lib/html-helpers.ts

```diff
@@ -82,6 +83,8 @@ export function getEmojiForStatus(status: MutantStatus) {
       return renderEmoji('👽', status);
     case MutantStatus.Timeout:
       return renderEmoji('⌛', status);
+    case MutantStatus.Pending:
+      return renderEmoji('⏳', status);
```

Comment:
I think we should keep emojis unique. I had to zoom in to see that this emoji is different than the one for a timeout. Maybe we should change the timeout status emoji to: ⏰

Thoughts @hugo-vrijswijk, @rouke-broersma @richardwerkman  ?

Reply:
I asked AI, and it came up with 3 answers:

🕒 Hourglass with Flowing Sand: This emoji can indicate that something is in progress or waiting for a certain duration.
⏳ Hourglass Done: This emoji can imply that something is almost done or nearing completion.
⚠️ Warning Sign: This emoji can suggest that something is uncertain or risky and may require attention or caution.
You can choose the emoji that best suits your needs or preferences. 😊

I think I like 🕒 the best

Reply:
How about ❔

Reply:
I like ⏰ for timeout. And ⏳ for pending

Reply:
Or perhaps no emoji and no presence in the filters at all? Are they actually relevant as a filter?

Reply:
I think they are, otherwise, they are not counted in the total number of mutants right?

Reply:
Why would they not be counted just because they don't have an emoji and aren't used as a filter?

Reply:
Should we still change the current emojis to something else?

Reply:
@xandervedder let's create a poll or something with the different options so we can make a decision.

Reply:
I will make one in the Stryker slack (general)